turbo.json

```diff
@@ -11,6 +11,7 @@
     "tools/bump-pack/**",
     "!tools/bump-pack/node_modules/**"
   ],
+  "globalEnv": ["CI"],
```

Comment:
Why is this needed?

Reply:
hangar makes use of this env variable to determine if it uses symlinks or if it'll install via tarballs. This ended up not mattering for testing this PR but it made sense to keep around anyways.

I should add this to hangar's config rather than the global one, so I'll change that.